src/unicode/Makefile.am

```diff
@@ -0,0 +1,37 @@
+EXTRA_DIST = \
+	make_widths.py edit_widths.py
+
+CLEANFILES = UCD.zip UnicodeData.txt PropList.txt EastAsianWidth.txt eaw_narrow_map eaw_wide_map eaw_delta_map
+
+noinst_SCRIPTS=make_widths.py
+
+# This is a set of developer-only targets for rebuilding Unicode
+# information in Mosh.
+# They're not intended to be run as part of normal automake processing.
+# Dependencies which must be provided manually: git python3 unzip
+
+PYTHONPATH=$(PWD)/google-libapps/libdot/third_party/wcwidth/:$(PWD)
+
+UCD.zip:
+	curl -O https://www.unicode.org/Public/11.0.0/ucd/UCD.zip
+
+google-libapps/libdot/third_party/wcwidth/ranges.py:
+	git submodule init
+	git submodule update
+
+ucd/PropList.txt ucd/UnicodeData.txt ucd/EastAsianWidth.txt: UCD.zip
```

Comment:
This target will run multiple times concurrently and race with itself when building with `-j`.